paddle/api/Matrix.cpp

```diff
@@ -52,6 +52,21 @@ Matrix* Matrix::createDense(const std::vector<float>& data, size_t height,
   return m;
 }
 
+Matrix* Matrix::createDenseFromNumpy(float* data, int dim1, int dim2,
+                                      bool copy, bool useGpu)
+                                      throw (UnsupportError) {
+  if (useGpu) {
+    /// Gpu mode only supports copy=True
+    if (!copy) {
+      UnsupportError e;
```

Comment:
Should include some message in UpsupportError(), otherwise the user will have hard to figure what's wrong.